packages/zeebe-element-templates-json-schema/test/fixtures/placeholder.js

```diff
@@ -0,0 +1,37 @@
+export const template = {
+  name: 'Tooltip',
+  id: 'example.com.tooltip',
+  appliesTo: [
+    'bpmn:ServiceTask'
+  ],
+  properties: [
+    {
+      label: 'Input with placeholder',
+      type: 'String',
+      binding: {
+        type: 'property',
+        name: 'prop'
+      },
+      placeholder: 'This field has a placeholder'
+    },
+    {
+      label: 'Input with placeholder',
+      type: 'Text',
+      binding: {
+        type: 'property',
+        name: 'prop'
+      },
+      placeholder: 'This field has a placeholder'
+    },
+    {
+      label: 'Input with placeholder',
```

Comment:
Do I remember correctly that we assume a string/text type in these cases?

Reply:
Yes, you do. This is why placeholder should be supported in such properties.